src/test/resources/regressions/features/arrays/array-literal-fail2.gobra

```diff
@@ -1,9 +0,0 @@
-// Any copyright is dedicated to the Public Domain.
-// http://creativecommons.org/publicdomain/zero/1.0/
-
-package pkg
-
-func foo() {
-	//:: ExpectedOutput(type_error)
-	a := [2]int{42}
```

Comment:
Is this now  a supported literal?

Reply:
Yes, `[2]int{42}` would be the same as `[2]int {42, 0}`.